platforms-wg/glossary/_index.md

```diff
@@ -5,44 +5,62 @@ description: "A collection of terms used in the Platform Working Group published
 
 See also <https://glossary.cncf.io/>.
 
-A **platform** aggregates capabilities to serve developers and operators in
+## Platform
+A platform aggregates capabilities to serve developers and operators in
 development and delivery of products, services and apps. In reference to the
 scenarios it aims to support, a platform may be named a "Developer Platform", a
 "Delivery Platform", an "App Platform" or even a "Cloud Platform." The
 connotations of the older term "Platform-as-a-Service", or PaaS, are also
 influential.
+_See also [Platforms](#platforms)_
 
-**Platforms** enable developers and operators to deliver applications and
-services faster by providing and managing common capabilities. Platforms bridge
-between platform users and platform capability providers, and are built and
-maintained by platform teams.
+## Platforms
+Platforms enable developers and operators to deliver applications and
+services faster by providing and managing common capabilities. It is common for
+organisations to have more than one platform though it can be useful to maintain
+similar user experience across them.
+_See also [Platform](#platform)_
 
-**Platform capability providers** develop and maintain the capabilities offered
+## Platform capability providers
+The companies and people who develop and maintain the capabilities offered
 by the platform. Providers can be both external organizations or internal teams,
 and capabilities can be infrastructure, runtime, or other supporting services.
 
-**Platform engineers** are responsible for developing and 
+## platform engineering
+An explicit form of DevOps cooperation between teams
+where a team offers an internal service as a product. This focus on providing
+a service means platform engineering requires investment in all parts of building
+platforms and their capabilities — the people, processes, policy, technology,
+and business outcomes.
+
+## Platform engineers
+The role who is focused on developing and 
 maintaining interfaces and tools to enable provisioning and integration of platform 
 capabilities in applications, according to the requirements and instructions provided 
 by platform product managers. Platform developers are usually grouped in platform teams.
 
-**Platform product managers** are responsible for understanding the experience of
+## Platform product managers
+The role who is chiefly responsible for understanding the experience of
```

Comment:
Perhaps take out the word who, so "A role chiefly responsible for understanding...."